package.json

```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/vieron/stylelint-webpack-plugin#readme",
   "peerDependencies": {
-    "stylelint": "^7.0.1"
+    "stylelint": "^7.0.1",
+    "webpack": "^2 || ^2.2.0-rc.0 || ^2.1.0-beta.0 || ^1.13.2"
```

Comment:
I'd rather just have `^2.0.0 || ^1.13.2` here... I would think it's better to just "officially" support stable versions. With `npm` >=3 or `yarn` you won't get peerDeps installed anyway so you can still choose what version you want.